src/Template/Pages/downloads.ctp

```diff
@@ -46,6 +46,8 @@ $description = __('File description');
 // Field names
 /* @translators: field name in Fields and structure on Downloads page */
 $sentence_id = __('Sentence id');
+/* @translators: field name in Fields and structure on Downloads page */
+$sentence_based_on_id = __('Sentence Id of base sentence');
```

Comment:
Style nitpick: All the other field name descriptions use `id` (lowercase letters) so I suggest using `Sentence id of base sentence`.